packages/tsconfig-reference/copy/en/options/noUnusedParameters.md

```diff
@@ -13,3 +13,12 @@ const createDefaultKeyboard = (modelID: number) => {
   return { type: "keyboard", modelID: defaultModelID };
 };
 ```
+
+Parameters declaration with names starting with an underscore _ are exempt from the unused parameter checking. e.g.:
+
+```ts twoslash
+function returnNull(_a) {
```

Comment:
This needs a type annotation to prevent the error: https://victorious-plant-05c166c10-3178.centralus.5.azurestaticapps.net/tsconfig/#noUnusedParameters

Reply:
Also, set `// @noUnusedParameters` here. Best to copy the above example and just rename the parameter, actually.

Reply:
Will do!